### PR TITLE
Fix bug in clear messages method

### LINKED ISSUE #1760: Desktop.clear_messages()
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Desktop.clear_messages() doesn't clear the message window.

Method clear_messages Failed:  Attribute Error. Please Check again
'Desktop' object has no attribute '_desktop'

image

### Steps To Reproduce

`desktop = pyaedt.Desktop('2022.2')`
`project_name = 'icepak_design.aedt'`
`ipk = pyaedt.Icepak(project_name)`
`desktop.clear_messages()`

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

aiohttp==3.8.1
aiosignal==1.2.0
altair==4.2.0
anyio==3.6.1
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.8
async-timeout==4.0.2
attrs==22.1.0
Babel==2.10.3
backcall==0.2.0
beautifulsoup4==4.11.1
bleach==5.0.1
blinker==1.5
cachetools==5.2.0
certifi==2022.6.15.1
cffi==1.15.1
charset-normalizer==2.1.1
click==8.1.3
clr-loader==0.1.7
colorama==0.4.5
commonmark==0.9.1
cPython==0.0.6
cycler==0.11.0
debugpy==1.6.3
decorator==5.1.1
defusedxml==0.7.1
diversipy==0.8
doepy==0.0.1
entrypoints==0.4
et-xmlfile==1.1.0
executing==1.0.0
fastjsonschema==2.16.1
Flask==2.2.2
fonttools==4.37.1
frozenlist==1.3.1
gitdb==4.0.9
GitPython==3.1.27
idna==3.3
imageio==2.21.3
importlib-metadata==4.12.0
ipykernel==6.15.2
ipython==8.5.0
ipython-genutils==0.2.0
itsdangerous==2.1.2
jedi==0.18.1
Jinja2==3.1.2
joblib==1.1.0
json5==0.9.10
jsonschema==4.16.0
jupyter-core==4.11.1
jupyter-server==1.18.1
jupyter_client==7.3.5
jupyterlab==3.4.7
jupyterlab-pygments==0.2.2
jupyterlab_server==2.15.1
kiwisolver==1.4.4
lxml==4.9.1
MarkupSafe==2.1.1
matplotlib==3.5.3
matplotlib-inline==0.1.6
mistune==2.0.4
multidict==6.0.2
nbclassic==0.4.3
nbclient==0.6.8
nbconvert==7.0.0
nbformat==5.5.0
nest-asyncio==1.5.5
notebook==6.4.12
notebook-shim==0.1.0
numpy==1.23.3
openpyxl==3.0.10
packaging==21.3
pandas==1.4.4
pandocfilters==1.5.0
parso==0.8.3
patsy==0.5.2
pickleshare==0.7.5
Pillow==9.2.0
plumbum==1.7.2
prometheus-client==0.14.1
prompt-toolkit==3.0.31
protobuf==3.20.1
psutil==5.9.2
pure-eval==0.2.2
pyaedt==0.6.0
pyarrow==9.0.0
pycparser==2.21
pydeck==0.8.0b3
pyDOE==0.3.8
Pygments==2.13.0
pymongo==4.2.0
Pympler==1.0.1
pyparsing==3.0.9
pyrsistent==0.18.1
python-dateutil==2.8.2
pythonnet==3.0.0rc4
pytz==2022.2.1
pytz-deprecation-shim==0.1.0.post0
pyvista==0.36.1
pywin32==304
pywinpty==2.0.7
pyzmq==23.2.1
requests==2.28.1
rich==12.5.1
rpyc==5.2.3
scikit-learn==1.1.2
scipy==1.9.1
scooby==0.6.0
semver==2.13.0
Send2Trash==1.8.0
six==1.16.0
sklearn==0.0
smmap==5.0.0
sniffio==1.3.0
soupsieve==2.3.2.post1
stack-data==0.5.0
statsmodels==0.13.2
streamlit==1.13.0
terminado==0.15.0
threadpoolctl==3.1.0
tinycss2==1.1.1
toml==0.10.2
tomli==2.0.1
toolz==0.12.0
tornado==6.2
traitlets==5.4.0
typing_extensions==4.3.0
tzdata==2022.2
tzlocal==4.2
urllib3==1.26.12
validators==0.20.0
vtk==9.2.0rc2
watchdog==2.1.9
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.4.1
Werkzeug==2.2.2
wslink==1.8.2
xmltodict==0.13.0
yarl==1.8.1
zipp==3.8.1

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Close #1760

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
Thanks @Samuelopez-ansys for fixing it.
